roles/reproducer/tasks/ci_job.yml

```diff
@@ -35,20 +35,23 @@
           ansible.builtin.copy:
             src: "{{ _reproducer_basedir }}/parameters/"
             dest: "/home/zuul/{{ job_id }}-params"
+            mode: "0775"
```

Comment:
i think all the tasks in this ci_job.yml are for files not directories so is the 0775 intentional here?

Reply:
+1